docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,248 @@ The following table shows how risk levels are applied to a host, based on the no
-. In the {security-app}, go to *Dashboards* -> *Overview*, then locate the *Current host risk scores* card in the lower-right corner.
-+
 [role="screenshot"]
-image::images/host-risk-score-enable-dev-tools.png[]
-. Click *Enable via Dev Tools*. This will open the Dev Tools Console in {kib}, which will be pre-populated with the scripts required to enable the host risk score feature.
-. Click the *Play* button to run each of the scripts, in their numbered order from 1 to 11.
-+
+image::images/enable-hrs.png[Enable Host Risk Score button]
+
+To enable host risk score from a host's details page: 
+
+. Go to *Explore* -> *Hosts*. 
+. Select the *All hosts* tab, then click a host name. 
+. On the details page, scroll down to the data tables, then select the *Host risk* tab. 
+. Click *Enable* to install the module. 
+
 [role="screenshot"]
-image::images/host-risk-score-dev-tools-console.png[]
+image::images/enable-hrs-details-pg.gif[Enable host risk score from the host's details page]
+
+To enable host risk score from Console in {kib}, open a browser window and enter the following URL:
+
+[source,console]
+----------------------------------
+{KibanaURL}/s/{spaceID}/app/dev_tools#/console?load_from={KibanaURL}/s/{spaceID}/internal/risk_score/prebuilt_content/dev_tool/enable_host_risk_score
+----------------------------------
 
-[[import-host-risk-score-dashboard]]
+NOTE: If you have existing content in Console, scroll to the bottom, then paste the content. 
+
+[[upgrade-host-risk-score]]
 [discrete]
-=== Import the host risk score dashboard
+=== Upgrade host risk score 
 
-After you have set up the framework for the host risk score feature, import pre-built dashboards to analyze risky hosts in your environment:
+If you previously enabled host risk score and are upgrading to {stack} version 8.5 or later, there'll be an option to "upgrade" host risk score instead of enabling it. 
 
-. Navigate to the *Current host risk scores* card (*Dashboards -> Overview*).
-. Click *Import dashboard*.
-+
-[role="screenshot"]
-image::images/host-risk-score-import-dashboard.png[]
+Before upgrading, be mindful of the following: 
+
+* Previous host risk scores are deleted since the older data is not preserved, and new scores are created. However, if you want to retain the old host risk scores, you'll need to reindex the data _before_ upgrading the feature. Your data will then be stored in the `ml_host_risk_score_{spaceID}` and `ml_$host_risk_score_latest_{spaceID}` indices. To learn how to do this, refer to {ref}/docs-reindex.html[Reindex API]. 
+
+* You must edit your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and remove the `xpack.securitySolution.enableExperimental:['riskyHostsEnabled']` feature flag. 
+
+After this is done, you can proceed with upgrading the host risk score feature from any of the following places in the {security-app}:
+
+* The Entity Analytics dashboard 
```

Comment:
@angorayc @SourinPaul Same question as above- do we want upgrades to happen from multiple places?

Reply:
I think @machadoum has already removed the ability to enable risk scores from the `Host/ User` pages https://github.com/elastic/security-team/issues/5093#issuecomment-1274544910. Maybe the change has not been incorporated in the docs yet. @machadoum @ferenrigue please confirm,

Reply:
On 8.5 we introduced the Enable/Upgrade callout that is displayed on Entity Analytics, Host, and User page.
<img width="1520" alt="Screenshot 2022-10-20 at 09 52 55" src="https://user-images.githubusercontent.com/1490444/196889920-ee4c2b79-acfc-4f82-88d1-fc3f9a686fb7.png">
<img width="1520" alt="Screenshot 2022-10-20 at 09 53 05" src="https://user-images.githubusercontent.com/1490444/196889924-9e8efa9f-6877-435a-b0b5-507fabca2b40.png">

<img width="1524" alt="Screenshot 2022-10-20 at 09 52 44" src="https://user-images.githubusercontent.com/1490444/196889914-8c5e0ea5-93b7-48ef-9c84-66ce7cdd8dfc.png">

Reply:
If I understand it correctly, the issue above is to remove the callout, so users should still be able to enable / upgrade risk score from multiple pages in 8.5